.zenodo.json

```diff
@@ -135,6 +135,10 @@
             "affiliation": "Eberhard Karls Universität Tübingen",
             "name": "Büchau, Yann Georg",
             "orcid": "0000-0003-2472-2051"
+        },
+        {
+            "name": "Ikko Ashimine",
+            "orcid": "0000-0002-3576-6677"
```

Comment:
may be with the next contribution `con/tributors` would add you and then we could add orcid ;)